web/template/layout/login-header.gohtml

```diff
@@ -92,7 +92,7 @@
         <div class="govuk-grid-column-two-thirds">
           <h2 class="service-header__heading">{{ tr .App "serviceName" }}</h2>
         </div>
-        <div class="govuk-grid-column-one-third">
+        <div class="govuk-grid-column-one-third app-hide-print">
```

Comment:
it might look better hiding the top-level govuk header too, so we have the title of the service at the top of the page?

Reply:
I was on the fence with this - having govuk there gives it a bit of official-ness but agree the onelogin stuff isn't needed. I'm happy to remove for now and we can see what people think in testing.

Reply:
The thing is we don't want people to think this is official or can be used as proof of something. It is just for record keeping so someone can remember they did a thing at a time

Reply:
Excellent point, I'd forgotten about that.